pipeline_dp/private_beam.py

```diff
@@ -142,16 +142,7 @@ def expand(self, pcol: pvalue.PCollection) -> pvalue.PCollection:
         backend = _get_beam_backend()
         dp_engine = pipeline_dp.DPEngine(self._budget_accountant, backend)
 
-        params = pipeline_dp.AggregateParams(
-            noise_kind=self._variance_params.noise_kind,
-            metrics=[pipeline_dp.Metrics.VARIANCE],
-            max_partitions_contributed=self._variance_params.
-            max_partitions_contributed,
-            max_contributions_per_partition=self._variance_params.
-            max_contributions_per_partition,
-            min_value=self._variance_params.min_value,
-            max_value=self._variance_params.max_value,
-            pre_threshold=self._variance_params.pre_threshold)
+        params = self._variance_params.to_aggregate_params()
```

Comment:
Nit (=style improvement) No need to have a local variable, since now it's just a one line (before this refactoring it was a long expressions, that's why it was  a separate variable)

And in other places.